supersim.go

```diff
@@ -30,21 +31,74 @@ var genesisL1JSON []byte
 //go:embed genesis/genesis-l2.json
 var genesisL2JSON []byte
 
+var (
+	DefaultAccounts       uint64                   = 10
+	DefaultMnemonic       string                   = "test test test test test test test test test test test junk"
+	DefaultDerivationPath hdaccount.DerivationPath = hdaccount.DefaultRootDerivationPath
+)
+
 var DefaultConfig = Config{
-	l1Chain:  Chain{anvilConfig: anvil.Config{ChainId: 1, Port: 8545, Genesis: genesisL1JSON}, opSimConfig: op_simulator.Config{Port: 8546}},
-	l2Chains: []Chain{{anvilConfig: anvil.Config{ChainId: 10, Port: 9545, Genesis: genesisL2JSON}, opSimConfig: op_simulator.Config{Port: 9546}}, {anvilConfig: anvil.Config{ChainId: 30, Port: 9555, Genesis: genesisL2JSON}, opSimConfig: op_simulator.Config{Port: 9556}}},
+	l1Chain: Chain{
```

Comment:
the config is a mess rn - we should add better support for flag parsing + applying defaults in a separate PR

Reply:
https://github.com/ethereum-optimism/supersim/issues/40